certif/app/routes/session-supervising.js

```diff
@@ -1,7 +1,24 @@
 import Route from '@ember/routing/route';
+import ENV from 'pix-certif/config/environment';
 
 export default class SessionSupervisingRoute extends Route {
   model(params) {
-    return this.store.queryRecord('session-for-supervising', { sessionId: params.session_id });
+    const sessionForSupervising = this.store.queryRecord('session-for-supervising', { sessionId: params.session_id });
+
+    this.poller = setInterval(
+      function (store) {
+        store.queryRecord('session-for-supervising', { sessionId: params.session_id });
+      },
+      ENV.APP.sessionSupervisingPollingRate,
+      this.store
+    );
```

Comment:
Avec une arrow function, on peut utiliser directement this.store
```suggestion
    this.poller = setInterval(
      () =>  this.store.queryRecord('session-for-supervising', { sessionId: params.session_id }),
      ENV.APP.sessionSupervisingPollingRate,
    );
```

Reply:
Autre possibilité, sortir ca dans un mini-utilitaire, au cas ou on en aurait besoin par la suite
````
const createPoller = (callback, ...args) => setInterval(
      callback,
      ENV.APP.sessionSupervisingPollingRate,
      ...args,
    );
````

Reply:
Done avec une arrow function